src/Client/Pages/ProtocolTemplates/ProtocolState.fs

```diff
@@ -70,7 +70,7 @@ module Protocol =
             let nextState = {
                 state with
                     TemplatesSelected   = templates
-                    WidgetTypes         = Routing.WidgetTypes.ProtocolSearch
+                    IsProtocolSearch    = true
```

Comment:
This line is not required, correct? As you would already be on the search screen? Remove it if it is not necessary to allow reusing this Msg in different logic approaches without switching to search view

Reply:
fixed